src/lib/cucumberjs/cucumber.js

```diff
@@ -79,8 +79,7 @@ class Cucumber {
       if (!this.cucumberChild.stopping) {
         log.debug('[chimp][cucumber] Cucumber not in a stopping state');
 
-        const result = jsonResults;
-        if (this.options.jsonOutput && JSON.parse(jsonResults).length) {
+        if (this.options.jsonOutput && jsonResults && JSON.parse(jsonResults).length) {
```

Comment:
After applying this changes manually on my 0.47.2 chimp version, I still get an error similar to #575 :

```
/home/tiagolr/ben/web/node_modules/chimp/dist/lib/cucumberjs/cucumber.js:102
          if (this.options.jsonOutput && jsonResults && JSON.parse(jsonResults).length) {
                          ^

TypeError: Cannot read property 'jsonOutput' of undefined
```

using the flags:
```
    screenshotsOnError: true,
    saveScreenshotsToDisk: true,
    saveScreenshotsToReport: true,
```